### PR TITLE
Update to 1.78.0 and add build for windows

### PR DESCRIPTION
**Pull request by adbedada**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by adbedada** [comment]
@mcs07 I gave myself a permission to use your recipe and attempt to update the on Conda-forge. 
I hope thats cool with you.

**Comment by adbeda** [comment]
@conda-forge/help-c-cpp can you help me fix the issue on the build for Windows?

**Comment by adbedada** [comment]
the error on Windows starts with

```
%SRC_DIR%>CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\..\..\VC\vcvarsall.bat" amd64  
CMake Error at CMakeLists.txt:41 (project):
  Failed to run MSBuild command:
```
I thought i noticed that you've created a patch in the past that fixed a similar issue but for `openjpeg`. do you an idea as to why the build is failing here?

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,22 @@
+mkdir build
+cd build
+
+cmake -G "%CMAKE_GENERATOR%" ^
```
Comment (xhochy) [inline recipe/bld.bat]: Use `NMake Makefiles` as the CMAKE Generator, this should fix the Azure issues

Comment (adbedada) [reply recipe/bld.bat]: @xhochy, thanks.
done and ready for review

**Comment by adbeda** [comment]
@conda-forge-admin, please rerender

**Comment by adbedada** [comment]
@conda-forge/help-c-cpp this recipe is ready for review.

Thanks!

**Comment by ocefpaf** [comment]
@adbeda consider adding yourself as a maintainer here.

**Pull request merged**